src/main/java/net/sf/ezmorph/object/DateMorpher.java

```diff
@@ -152,14 +154,13 @@ public Date getDefaultValue() {
 
     @Override
     public int hashCode() {
-        HashCodeBuilder builder = new HashCodeBuilder();
-        builder.append(formats);
-        builder.append(locale);
-        builder.append(lenient);
+        int result = Arrays.hashCode(formats);
+        result = 31 * result + Objects.hashCode(locale);
+        result = 31 * result + Boolean.hashCode(lenient);
         if (isUseDefault()) {
-            builder.append(getDefaultValue());
+            result = 31 * result + Objects.hashCode(getDefaultValue());
```

Comment:
37 is used here to keep the same logic as in `commons-lang`

Using a smaller number like 31 may result in more hash collisions.

But this may not be a big problem.

@basil